--- a/mlx_int.h
+++ b/mlx_int.h
@@ -125,8 +125,8 @@
 int				mlx_int_find_in_pcm();
 int				mlx_int_anti_resize_win();
 int				mlx_int_wait_first_expose();
-int				mlx_int_rgb_conversion();
-int				mlx_int_deal_shm();
+int				mlx_int_rgb_conversion(t_xvar *xvar);
+int				mlx_int_deal_shm(t_xvar *xvar);
 void			*mlx_int_new_xshm_image();
 char			**mlx_int_str_to_wordtab();
 void			*mlx_new_image();
